src/axom/core/tests/core_array.hpp

```diff
@@ -1286,7 +1286,7 @@ void checkIteratorDeviceImpl()
   axom::Array<int, 1, axom::MemorySpace::Host> v_int_host(SIZE);
   axom::Array<int, 1, axom::MemorySpace::Device> v_int(SIZE);
 
-  auto v_int_view = v_int.view();
+  //auto v_int_view = v_int.view();
```

Comment:
If not needed, should this line be removed?

Reply:
Done.